src/libraries/System.Runtime.InteropServices/gen/Microsoft.Interop.SourceGeneration/TypeSymbolExtensions.cs

```diff
@@ -30,10 +30,10 @@ public static bool IsConsideredBlittable(this ITypeSymbol type)
         {
             unsafe
             {
-                return IsBlittableWorker(type, ImmutableHashSet.Create<ITypeSymbol>(SymbolEqualityComparer.Default), &IsConsideredBlittableWorker);
+                return IsBlittableWorker(type, ImmutableHashSet.Create<ITypeSymbol>(SymbolEqualityComparer.Default), compilation: null, &IsConsideredBlittableWorker);
```

Comment:
I thought we had nullability checks enabled. Did we turn them off?

Reply:
They've been acting really weird ever since we moved into dotnet/runtime. I'll see if I can get them re-enabled.

Reply:
dotnet/runtime libraries adds `<NoWarn>$(NoWarn);nullable</NoWarn>` to any project that doesn't target .NETCoreApp.

Looks like all other source generators in the repo multi-target to get nullable warnings and ensure the warnings from the BCL types are accurate (and then only ship the .NET Standard 2.0 assembly)